### ISSUE TITLE
Mentor Registration for December 2nd, 2017

### ISSUE THREAD

**Issue by Xabadu** [nodeschool/sanfrancisco#228]
Mixmax is hosting us on December 2nd from 1pm - 5pm. The address is: 512 2nd St, San Francisco, CA 94107 -- Comment to get involved or ask questions.

Mentors should:
- Have already completed the [core NodeSchool workshoppers](http://nodeschool.io/#workshopper-list) before the event.
- Having also completed elective workshoppers is a great bonus._
- Review the [Mentor Best Practices](https://github.com/nodeschool/organizers/wiki/Event-Mentor-Best-Practices) wiki page.
- Arrive 30 minutes early (12:30p) so we can get setup and have our mentor meeting.

Confirmed mentors: 
- [x] @mattiaerre
- [x] @benhalverson  
- [x] @bengl 
- [x] @cantremember
- [x] @itsmed  
- [x] @n0f3 
- [x] @fforres 
- [x] @iqnivek 
- [ ] You? 
- [ ] You?

Organizers:
- [x] @nebrius
- [x] @tlhunter
- [x] @Xabadu

Attendee registration link: https://www.meetup.com/sfnodeschool/events/245191260/

Slides:
 - Intros: TBD
 - Shoutouts: TBD
 - Shoutout loop: TBD

**Comment by Trott** [comment]
o/

**Comment by Xabadu** [comment]
@samarthgulati @bengl @benhalverson @n0f3 @mattiaerre @shiya @iqnivek 👍 ? 🍕 ?  😎 ?

**Comment by mattiaerre** [comment]
@Xabadu I'm down!
-Mattia

**Comment by benhalverson** [comment]
Yep I can make it

**Comment by bengl** [comment]
Very in.

**Comment by nebrius** [comment]
cc @BerkeleyTrue

**Comment by cantremember** [comment]
```
   _.-._
  | | | |_
  | | | | |
  | | | | |
_ |  '-._ |
\`\`-.'-._;
 \    '   |
  \  .`  /
   |    |
```

**Comment by shiya** [comment]
I'm in!

**Comment by n0f3** [comment]
always available 💯

**Comment by fforres** [comment]
Ill be around for that day! I'd love to help.
Count me in

**Comment by iqnivek** [comment]
I'll join too!

**Comment by skipzero** [comment]
I'm available to mentor as well. Need to get my JS fix! :)

**Comment by Xabadu** [comment]
@Trott @mattiaerre @benhalverson @bengl @cantremember @shiya @n0f3 @fforres @iqnivek @skipzero
 
Mentors! Heads up: We have a venue change for this Saturday. We'll be going to Mixmax (512 2nd St) instead of Google Launchpad.

Our apologies for any inconvenience that this may cause to any of you. If for any reason you can't attend due to this change, we completely understand.

If you have any questions, don't hesitate to ask!

**Comment by bengl** [comment]
![](https://media.giphy.com/media/xT8qBrtlhoV1wl7yUg/giphy.gif)

**Comment by shiya** [comment]
Hey everyone, I have a family emergency for which I have to fly home last minute. I'm going to have to miss this one. Sorry :( Y'all have fun!

**Comment by itsmed** [comment]
It's been a while, but I'll be there this time
<img src="https://tenor.com/view/iam-back-im-back-back-dog-bulldog-gif-7751662" />

**Comment by skipzero** [comment]
i’ve got to bow out this time, sorry. looks like you won’t be short handed without me tho! ill see y’all in the new year!

**Comment by Trott** [comment]
Sorry to say that it looks like I'll be missing this one after all.

**Comment by Xabadu** [comment]
Thanks for letting us know @shiya @skipzero @Trott

**Comment by Xabadu** [comment]
Hey mentors!

Thanks again for volunteering tomorrow, we really appreciate it! Please remember to be at the venue a bit before 1 pm (ideally around 12.45), so we can have a small meeting and check-in.

See you all tomorrow!

**Comment by bengl** [comment]
#### TL;DR Version:

Hey let's go to breakfast tomorrow: **[Mission's Kitchen](https://www.yelp.com/biz/missions-kitchen-san-francisco) at 11:00AM**

#### Long version:

Hello mentors!

I have an excellent idea! (I recognize this might be too late notice for some people, sorry!)

Let's get a super-delicious diner breakfast/brunch before we head out and mentor! Wouldn't that be a pretty sweet tradition to start?

I'll be heading to  **[Mission's Kitchen](https://www.yelp.com/biz/missions-kitchen-san-francisco) at 11:00AM**, in search of eggs, bacon, toast and coffee, which should leave plenty of time to order, eat, pay and leave to get to Mixmax well before 12:30. **All mentors are invited!** I understand that that's a little far from the venue, but I'll grab a Lyft to Mixmax for as many folks show up (or can fit into the car).

*Note: I'm up for alternative diners if someone has a preference, I just picked Mission's Kitchen because I've tried a lot of diners in SF before but never that one!*

**Comment by benhalverson** [comment]
@Xabadu I'm just now leaving from Santa Clara.

**Comment by Xabadu** [comment]
Thank you all for coming today. It was a great event!

See you in January! 🤘
